velox/exec/tests/utils/PlanBuilder.h

```diff
@@ -257,6 +257,23 @@ class PlanBuilder {
       const std::string& outputDirectoryPath,
       dwio::common::FileFormat fileFormat = dwio::common::FileFormat::DWRF);
 
+  /// Adds a TableWriteNode to write certain input columns into an
+  /// un-partitioned un-bucketed Hive table. Specify the column and aggregation
+  /// function to collect column statistics.
+  ///
+  /// @param inputColumns Input columns to write.
+  /// @param outputDirectoryPath Path to a directory to write data to.
+  /// @param fileFormat File format to use for the written data.
+  /// @param statsAggField Column to collect statistics during write.
+  /// @param statsAggFunction Aggregation function name for column statistics
+  /// collection during write.
+  PlanBuilder& tableWrite(
+      const RowTypePtr& inputColumns,
```

Comment:
Isn't `inputColumns` always the same as the previous plan node's output type (`planNode_->outputType()`)? The two tests in its PR seem to be so.

Reply:
`inputColumns` is a parameter in `SharedArbitratorTest`'s `createInsertPlan` utility function, unless we change the signature of `createInsertPlan`?